### PR TITLE
[feature/RPA-511]: Open source section

### PR DESCRIPTION
**Pull request by Klau-Sam**
## Description

image
image
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-landing/translations/en/open-source.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "Landing.OpenSource.Heading.Description.Part.1": "Learn the",
+    "Landing.OpenSource.Heading.Description.Part.2": "undeniable advantages",
+    "Landing.OpenSource.Heading.Description.Part.3": "of open source software",
+    "Landing.OpenSource.Card.1.Title": "Greater security",
+    "Landing.OpenSource.Card.1.Text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    "Landing.OpenSource.Card.2.Title": "Free product",
+    "Landing.OpenSource.Card.2.Text": "",
+    "Landing.OpenSource.Card.3.Title.Part.1": "Easy to customize ",
+    "Landing.OpenSource.Card.3.Title.Part.2": "for your own needs",
+    "Landing.OpenSource.Card.3.Text": ""
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/translations/en/open-source.json]: why empty? xd

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceBackground/OpenSourceBackground.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react';
+
+import { OPEN_SOURCE_SECTION_ID } from '#src-landing/utils/utils';
+
+import styles from './OpenSourceBackground.module.scss';
+
+const OpenSourceBackground: FC = ({ children }) => (
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceBackground/OpenSourceBackground.tsx:7]: isnt there any TS error? im sure that FC from react 18 does not contain `children` by default

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceContent/OpenSourceContent.module.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
+@import '#src-landing/scss/main.scss';
+
+$ICON_SIZE: 76px;
+$SEPARATOR_FIRST_DIM: 2px;
+$SEPARATOR_SECOND_DIM: 36px;
+
+.content {
+    width: 100%;
+    max-width: 1672px;
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceContent/OpenSourceContent.module.scss]: why that size

**Review by preludium — DISMISSED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceContent/OpenSourceContent.module.scss (hunk 3 above)
Comment (Klau-Sam) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceContent/OpenSourceContent.module.scss]: from SectionContent.module.scss, so I'm just gonna use it xd

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceBackground/OpenSourceBackground.tsx (hunk 2 above)
Comment (Klau-Sam) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceBackground/OpenSourceBackground.tsx:7]: no, there isn't

**Review by preludium — APPROVED** [review]

**Pull request merged**